src/JBSwapTerminal.sol

```diff
@@ -554,7 +554,7 @@ contract JBSwapTerminal is JBPermissioned, Ownable, IJBTerminal, IJBPermitTermin
     {
         {
             // Check for a quote passed in by the user/client.
-            (bool exists, bytes memory quote) = JBMetadataResolver.getDataFor(bytes4("SWAP"), metadata);
+            (bool exists, bytes memory quote) = JBMetadataResolver.getDataFor(bytes4(bytes20(address(this))) >> 1, metadata);
```

Comment:
with a bit shift pattern, we're increasing the collision risk -> why not unchecked increment?